test/internet/test-http-dns-fail.js

```diff
@@ -5,34 +5,34 @@
  */
 
 const common = require('../common');
-var assert = require('assert');
-var http = require('http');
+const assert = require('assert');
+const http = require('http');
 
-var hadError = 0;
+let hadError = 0;
 
 function httpreq(count) {
   if (1 < count) return;
```

Comment:
Would you mind reversing the conditional here to `count > 1` while you're in this file? I personally find it much easier to understand when scanning code...